README.md

```diff
@@ -88,6 +88,16 @@ Under the run/ debug configuration drop down menu, click `Edit Configurations...
 
 See [PushNotifications.md](PushNotifications.md) for detailed information on getting PN working with the example app.
 
+#### [Important Note regarding Firebase and Push Notifications](https://github.com/ably/ably-flutter/issues/226)
```

Comment:
This readme actually has two sections with the same "Push Notifications" heading. 🙄 
[This one](https://github.com/ably/ably-flutter/tree/add-workaround_for_firebase_messaging#push-notifications), and [another one later on](https://github.com/ably/ably-flutter/tree/add-workaround_for_firebase_messaging#push-notifications-1).

Perhaps that was the initial impetus for the creation of the [dedicated, discrete document `PushNotifications.md`](https://github.com/ably/ably-flutter/blob/add-workaround_for_firebase_messaging/PushNotifications.md).

Regardless of reasons, the reality is that there are two places that could do with this information being visible... therefore my suggestion would be to move this new content to the dedicated document.

Reply:
I was aware of that but I thought this would be a bit more prominent if we have this on the original readme. And I also think that the readme here is in needs of shortening, Perhaps we can then move push notification section here. But I will move this to the separate file for now

Reply:
https://github.com/ably/ably-flutter/pull/289/commits/b72803873bccfe4099a92e0dd9fb999388f288ae